docs/docs/training-data-format.mdx

```diff
@@ -458,6 +493,35 @@ stories:
   - action: action_store_feedback
 ```
 
+If you want to put bot utterance directly in the stories, use `bot:` key followed by
+the text that you want your bot to say
+
+:::caution
+This feature is experimental.
+We introduce experimental features to get feedback from our community, so we encourage you to try it out!
+However, the functionality might be changed or removed in the future.
+If you have feedback (positive or negative) please share it with us on the [Rasa Forum](https://forum.rasa.com).
+
+:::
+
```

Comment:
Let's add a sentence that an end-to-end story might look like this. E.g. it can be a mixture of different keys.